Portal/src/Datahub.Portal/Pages/Landing/WelcomeBanner.razor

```diff
@@ -28,28 +30,23 @@ else
 @code {
 
     private string _userBackgroundUrl;
+    private string _greeting;
     private List<AnnouncementPreview> _previews = new();
     private bool _isLoading = true;
-    private DateTimeOffset _now = DateTimeOffset.Now;
+    private IJSObjectReference _module;
     private string _personalBackgroundStyle => $"background: linear-gradient(180deg, rgba(0, 0, 0, 0) 0%, black 100%), url('{GetBackground()}') no-repeat;background-size: cover; height: {AnnouncementCarousel.CarouselHeight}px;";
 
     protected override async Task OnInitializedAsync()
     {
         var portalUser = await _userInformationService.GetCurrentPortalUserAsync();
+        _module = await _jsRuntime.InvokeAsync<IJSObjectReference>("import", "./Pages/Landing/WelcomeBanner.razor.js");
```

Comment:
Need to dispose of this if we're going to reference it.

You can `@implements IAsyncDisposable` on the component and then dispose of it in `IAsyncDispoable.DisposeAsync`

see https://learn.microsoft.com/en-us/aspnet/core/blazor/javascript-interoperability/call-javascript-from-dotnet?view=aspnetcore-7.0#javascript-isolation-in-javascript-modules